printer_mods/Mellow/V0_Fly-Gemini/readme.md

```diff
@@ -0,0 +1,31 @@
+# V0 adaptation model of Fly-Gemini
+
+## Fixed bracket
+[Bracket](./Bracket)
+
+### Gemini_v0.1_Stent_FLYLOGO.STL
+
+- Use VHB tape to secure to the V0 backplane
+- The bracket and the motherboard are directly connected with 4 M3*6 or M3*8 screws
```

Comment:
use an x instead of *. The asterisk confuses the markup language. `M3x6 or M3x8`

Reply:
I was negligent, thanks